docs/src/content/docs/request-state.mdx

```diff
@@ -0,0 +1,59 @@
+---
+title: Request State
+description: Request State
+packageName: '@inox-tools/request-state'
+challenger: [Florian, Oliver]
+---
+
+Astro provides [`Astro.locals`/`context.locals`](https://docs.astro.build/en/reference/api-reference/#astrolocals) for shared state between different components being rendered in a page.
+
+This state can only be accessed from an Astro component using the `Astro` constant or from middlewares from the context. Sharing state between framework components is not provided and inconvenient.
```

Comment:
I think only mentioning framework components could be misleading. I think it would be more useful to say it's available in client side modules (or whatever it's called), *for example* framework components

Reply:
This is talking about the limitations of `Astro.locals` regarding framework components on the server. The part about state not being shared with the client is on the next paragraph.

I'll rewrite this introduction, it really is not very clear right now.